src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -277,6 +284,10 @@ def _configure_db_create_params(
     arg_ctx.argument('elastic_pool_id',
                      arg_type=elastic_pool_id_param_type,
                      help='The name or resource id of the elastic pool to create the database in.')
+    
+    arg_ctx.argument('compute_model',
+                     arg_type=compute_model_param_type,
+                     help='The compute model of this database. Currently we support Provisioned and Serverless.')
```

Comment:
`The compute model of the database. Allowed values include: Provisioned, Serverless.`

Reply:
I think you can put this help text into `compute_model_param_type` so that it's not repeated later.

Reply:
agreed!